rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -103,10 +103,23 @@ def _populate_direct_deployment_counts(deployment):
         return deployment
 
     @staticmethod
-    def _handle_deployment_labels(rm, deployment, raw_labels_list):
-        deployment_parents = deployment.deployment_parents
-        new_labels = rest_utils.get_labels_list(raw_labels_list)
+    def _update_deployment_counts(rm, deployment, new_labels):
+        if deployment.deployment_parents:
+            types = rm.get_deployment_object_type_from_labels(new_labels)
+            to_srv = deployment.is_environment and 'environment' not in types
+            to_env = not deployment.is_environment and 'environment' in types
+            _type = 'service' if to_srv else 'environment' if to_env else None
```

Comment:
How would this be `None`?

Reply:
it would be None like if did not change the the `csys-type-obj` then it should be None, that means we did not remove/add `csys-type-obj` to the deployment

Reply:
But, unless I'm missing something, it would always get a boolean value.

Reply:
_type will be one of three values: `service` or `environment` or `None` and `None` means nothing is need to do where no add/remove happened to the `csys-type-obj` label